models/articles-model.js

```diff
@@ -31,4 +31,25 @@ exports.selectAllArticles = () => {
       const articles = result.rows;
       return articles;
     });
-};
\ No newline at end of file
+};
+
+exports.updateArticleVotes = (articleId, votes) => {
+  if (typeof articleId !== "number" || typeof votes !== "number") {
```

Comment:
PSQL will have error codes for an article id that isn't a number _and_ the votes as the seed defines the column data type. There is no need for a custom reject here.

Reply:
<img width="812" alt="Screenshot 2024-08-27 at 14 17 32" src="https://github.com/user-attachments/assets/d8e4574e-a624-41c4-ad2e-f00eb83efee6">